src/code/z_collision_check.c

```diff
@@ -1,13 +1,22 @@
-#include "global.h"
+#include "gfx.h"
+#include "macros.h"
+#include "regs.h"
+#include "sfx.h"
+#include "sys_math3d.h"
+#include "sys_matrix.h"
 #include "terminal.h"
 #include "versions.h"
-
+#include "z64collision_check.h"
```

Comment:
This is my personal preference (and I have been trying to enforce it on MM), but what do you think on putting the system's header at the top of the file?

Reply:
seems like for now hes doing alphabetical. I think there are other orderings that make more sense, but this is prob fine for now. things are gonna change a bunch anyway when we finally remove "z64" from these names